app/__init__.py

```diff
@@ -1,7 +1,7 @@
 from flask import Flask
 from flask.ext.bootstrap import Bootstrap
 from config import config
-
+from main import main as main_blueprint
```

Comment:
I took a stab at fixing this and there is some import strangeness going on in the tests. Gonna take another look tomorrow.